lighthouse-viewer/app/src/viewer-ui-features.js

```diff
@@ -44,7 +42,7 @@ class ViewerUIFeatures extends ReportUIFeatures {
    * @override
    */
   getReportHtml() {
-    // @ts-ignore - TODO(bckenny): remove ignore when this.json is an LHR instead of ReportJSON.
+    // @ts-ignore - TODO(bckenny): remove ignore when this.json is an LHR instead of LH.ReportResult.
```

Comment:
everything extends LH.Result now, right? so we should be good to remove this?

Reply:
is this ignore still necessary? Since `LH.ReportResult` extends `LH.Result`, I would assume you could pass it into `ReportGenerator.generateReportHtml`?